k8s/config/crd/bases/k8s.v6d.io_sidecars.yaml

```diff
@@ -0,0 +1,128 @@
+---
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.9.2
+  creationTimestamp: null
+  name: sidecars.k8s.v6d.io
+spec:
+  group: k8s.v6d.io
+  names:
+    kind: Sidecar
+    listKind: SidecarList
+    plural: sidecars
+    singular: sidecar
+  scope: Namespaced
+  versions:
+  - additionalPrinterColumns:
+    - jsonPath: .status.current
+      name: Current
+      type: string
+    - jsonPath: .spec.replicas
+      name: Desired
+      type: string
+    name: v1alpha1
+    schema:
+      openAPIV3Schema:
+        properties:
+          apiVersion:
+            type: string
+          kind:
+            type: string
+          metadata:
+            type: object
+          spec:
+            properties:
+              etcdCmd:
+                default: etcd
+                type: string
```

Comment:
I think the option `etcd_cmd` shouldn't be exposed to operator. On kubernetes, we luanch the etcd pods/deployments for vineyard and vineyard itself won't have the chance/possibility to find the etcd binary and luanch a etcd server in its container.

And indeed that won't work.

Reply:
Make sense, I'll update it later.